apps/web/components/experiment-settings/experiment-settings.tsx

```diff
@@ -45,6 +45,11 @@ export function ExperimentSettings({ experimentId }: ExperimentSettingsProps) {
         />
       </div>
 
+      {/* Protocol Management - New Row */}
+      <div className="mt-2">
+        <ExperimentProtocolManagement experimentId={experimentId} />
```

Comment:
why not just add the mt-2 to ExperimentProtocolManagement instead of adding one more div